packages/api/src/parse-cli.ts

```diff
@@ -78,14 +80,19 @@ export type CliArgs = ReturnType<typeof parseCli>;
 // before passing it to yargs completely breaks type inference, which is a huge shame.
 // So... this monkeypatches yargs to capture that variable. If you know of a more
 // elegant way, I'd love to hear it!
-let args;
+let allOptions;
 const originalOpts = yargs.options;
-yargs.options = function (arg) {
-  args = arg;
-  return originalOpts.call(this, arg);
+yargs.options = function (...args) {
+  if (!allOptions) {
+    allOptions = args[0];
+  }
+  return originalOpts.call(this, ...args);
 };
 
 export default function parseCli(argv?: string | readonly string[]) {
+  if (!argv) {
+    argv = process.argv.slice(2);
```

Comment:
I thought `yargs` did this automatically

Reply:
Apparently not the way we're using it here. I'm not super clear why I had to make these changes but esbuild really didn't want to work with the way we were invoking yargs.